drl-exporter/.gitignore

```diff
@@ -0,0 +1 @@
+*.tgz
```

Comment:
A lot of files are missing "new lines at the end of file" - check why your editor does this, it would be neat to remove this warning.

Reply:
Should be sorted now. I'll keep an eye on that